client/src/components/sidebar/MobileMenuButton.tsx

```diff
@@ -0,0 +1,51 @@
+import MenuIcon from '@mui/icons-material/Menu';
+import { Tooltip } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+interface MobileMenuProps {
+  onClick: () => void;
+  toolTipTitle: string;
+}
+
+const StyledMobileMenu = styled('button')(({ theme }) => ({
+  position: 'fixed',
+  top: 0,
+  left: 0,
+  zIndex: 1000,
+  display: 'flex',
+  height: 40,
+  width: 40,
+  alignItems: 'center',
+  justifyContent: 'center',
+  padding: theme.spacing(1.5),
+  margin: 10,
+  marginRight: 0,
+  borderRadius: 8,
+  cursor: 'pointer',
+  transition: 'background-color 0.3s ease',
+  color: theme.palette.text.primary,
+  backgroundColor: theme.palette.background.paper,
+
+  // selected instead of hover so it stays opaque when hovering
+  '&:hover': {
+    backgroundColor: theme.palette.action.selected,
```

Comment:
suggestion: change to using
```suggestion
    backgroundColor: theme.palette.secondary.light,
```

this would result in it turning lightly grey but not completely transparent that doesn't do well when text is behind the button. See below:

<img width="308" alt="image" src="https://github.com/user-attachments/assets/cbfa051e-dfb6-4eb4-abc4-ef46f50ff411" />

<img width="310" alt="image" src="https://github.com/user-attachments/assets/fc05bca3-139f-430a-a917-e79c1d5227ea" />